### PR TITLE
Fix ConnectPassport for Symfony 5.4

### PR DESCRIPTION
**Pull request by javiereguiluz**
I'm trying to upgrade to Symfony 5.4 an application that uses Connect and I'm seeing this problem related to security:

connect-passport-error

The change proposed in this PR solved the problem for me and things seem to be working. However, I don't understand well this part of Symfony Security, so I'm really don't know what I'm doing. This change could be dangerous and cause problems.

Maybe @chalasr or @wouterj or @tgalopin can help us here. Thanks!

### CODE REVIEW

File: src/Security/ConnectPassport.php [hunk 1]
```diff
@@ -12,14 +12,15 @@
...
+use Symfony\Component\Security\Http\Authenticator\Passport\Passport;
 use Symfony\Component\Security\Http\Authenticator\Passport\PassportTrait;
 use Symfony\Component\Security\Http\Authenticator\Passport\UserPassportInterface;
 use SymfonyCorp\Connect\Api\Entity\User;
 
 /**
  * @author Fabien Potencier <fabien@symfony.com>
  */
-class ConnectPassport implements UserPassportInterface
+class ConnectPassport extends Passport implements UserPassportInterface
```
Comment (chalasr) [inline src/Security/ConnectPassport.php:23]: It should stop implementing the deprecated interface directly also, don't forget the `use` statement :)

```suggestion
class ConnectPassport extends Passport
```

**Review by chalasr — APPROVED** [review]
Looks good!

File: src/Security/ConnectPassport.php (hunk 1 above)
Comment (javiereguiluz) [reply src/Security/ConnectPassport.php]: Thanks for the review! I've removed the interface.

**Comment by wouterj** [comment]
We can deprecate `ConnectPassport` completely. After implementing it for this package, Fabien proposed (and merged) passport attributes in core, which removes the need for a custom passport class.

**Comment by fabpot** [comment]
Let's indeed remove this class and replace it with the simpler attributes feature.

**Comment by javiereguiluz** [comment]
I've tried to do the requested changes, please review it. Thanks!

**Comment by fabpot** [comment]
I suppose we need to bump some constraints in `composer.json`?

**Pull request merged**